public_html/wp-content/mu-plugins/latest-site-hints.php

```diff
@@ -0,0 +1,216 @@
+ *
+ * This helps search engines know to direct queries for "WordCamp Seattle" to `seattle.wordcamp.org/2020`
+ * instead of `seattle.wordcamp.org/2019`, even if `/2019` has a higher historic rank.
+ */
+function canonical_link_past_home_pages_to_current_year() {
+	// We don't want to penalize historical content, we just want to boost the new site.
+	if ( ! is_front_page() ) {
+		return;
+	}
+
+	$latest_site = get_latest_site();
+	if ( ! $latest_site ) {
+		return;
+	}
+
+	// Remove default canonical link, to avoid duplicates.
+	// @todo: This will need to be updated if rel_canonical_link() is ever merged to Core.
+	remove_action( 'wp_head', 'WordPressdotorg\SEO\Canonical\rel_canonical_link' );
+
+	printf(
+		'<link rel="canonical" href="%s" />' . "\n",
+		esc_url( set_url_scheme( trailingslashit( '//' . $latest_site->domain . $latest_site->path ) ) )
+	);
+}
+
+/**
+ * If notification is shown, misuse body.admin-bar class as most of the WordCamp themes already that that into
+ * account. This makes its a whole lot easier to place the notification on top of contens.
+ */
+function add_notification_classes_to_body( $classes ) {
+	$classes[] = 'admin-bar'; // Add this as most of the current WordCamp themes already take the adminbar into account.
+	return $classes;
+}
+
+/**
+ * Simple styles for the notification.
+ */
+function add_notification_styles() { ?>
+	<style type="text/css">
+		html {
+			margin-top: 35px !important;
+		}
+
+		.wordcamp-latest-site-notify {
+			background: #1d2327;
+			text-align: center;
+			padding-top: 5px;
+			font-size: 15px;
+			height: 35px;
+			position: fixed;
+			top: 0;
+			left: 0;
+			width: 100%;
+			min-width: 600px;
+			z-index: 99999;
+		}
+
+		.wordcamp-latest-site-notify p,
+		.wordcamp-latest-site-notify a {
+			color: #f0f0f1;
+			margin: 0;
+		}
+
+		.wordcamp-latest-site-notify a {
+			font-weight: 600;
+		}
+
+		.wordcamp-latest-site-notify a:hover,
+		.wordcamp-latest-site-notify a:active {
+			color: #72aee6;
+		}
+	</style>
+<?php }
+
+/**
+ * Show the actual notification containing link to latest site to user.
+ */
+function show_notification_about_latest_site() {
+	global $current_blog;
+
+	$latest_site = get_latest_site();
+	if ( ! $latest_site ) {
+		return;
+	}
+
+	echo '<div id="wpadminbar" class="wordcamp-latest-site-notify">
+		<p>' . esc_html( get_blog_details( $current_blog->blog_id )->blogname ) . ' is over. Check out <a href="' . esc_url( get_site_url( $latest_site->blog_id ) ) . '">' . esc_html( get_blog_details( $latest_site->blog_id )->blogname ) . '</a>!</p>
+	</div>';
+}
+
+/**
+ * Get the latest site WP_Site object.
+ *
+ * @return bool|object
+ */
+function get_latest_site() {
+	global $current_blog;
+
+	$latest_site_id = get_latest_site_id( $current_blog->domain, $current_blog->path );
+
+	if ( ! $latest_site_id ) {
+		return false;
+	}
+
+	$latest_site = get_site( $latest_site_id );
+
+	if ( $latest_site->domain === $current_blog->domain ) {
```

Comment:
This needs to check the `path` as well, since the year is now there. It's probably easier to compare the `blog_id` though.